src/main/kotlin/com/hedvig/underwriter/service/model/QuoteSchema.kt

```diff
@@ -89,4 +89,15 @@ sealed class QuoteSchema {
         @JsonSchema(title = "Is Youth", required = true)
         val isYouth: Boolean
     ) : QuoteSchema()
+
+    data class DanishHomeContents(
```

Comment:
Shall we have the same name as the NorwegianHomeContent ? I mean without `s`

```suggestion
    data class DanishHomeContent(
```